src/utils/Services.ts

```diff
@@ -196,6 +197,17 @@ export function getStudentServices(
       badgeFunction: (dashboard: FullDashboardType): number =>
         dashboard.available_tutorials,
     },
+    {
+      key: SERVICES_KEY.USEFUL_LINKS,
+      title: 'Liens utiles',
+      subtitle: i18n.t('screens.services.descriptions.usefulLinks'),
+      image: Urls.images.wiketud,
```

Comment:
I propose we copy the `Wiketud.png` image on `etud` to something like `LiensUtiles.png` (see [urls](/ClubInfoInsaT/application-amicale/blob/master/src/constants/Urls.tsx) ). That way we can later easily change the image without updating the app.

Reply:
Could you please use `https://etud.insa-toulouse.fr/~amicale_app/images/UsefulLinks.png` instead please ?